test cases/frameworks/7 gnome/resources-data/meson.build

```diff
@@ -0,0 +1,16 @@
+subdir('subdir')
+
+fake_generator_script = '''
+import os, sys
+assert os.path.exists(sys.argv[1]), "File %s not found" % sys.argv[1]
+print("This is a generated resource")
+'''
+
+# Generate file res3.txt from file res3.txt.in. This is then included
+# in a GResource file, driven by resources/meson.build.
+res3_txt = custom_target('res3.txt',
+  input: 'res3.txt.in',
+  output: 'res3.txt',
+  command: ['python', '-c', fake_generator_script, '@INPUT@'],
```

Comment:
This invokes Python 2 (on some machines). Our unit tests always use `python3` explicitly rather than just `python`. This will change once #777 lands but for now just use `python3`.